packages/amplify-category-function/provider-utils/awscloudformation/index.js

```diff
@@ -315,27 +315,32 @@ async function openEditor(context, category, options) {
 async function invoke(context, category, service, resourceName) {
   const { amplify } = context;
   serviceMetadata = amplify.readJsonFile(`${__dirname}/../supported-services.json`)[service];
-  const { inputs } = serviceMetadata;
+  const { inputs: rawInputs } = serviceMetadata;
+  const inputs = {};
+  rawInputs.forEach(input => {
+    inputs[input.key] = input;
+  });
+  const { handlerFilename, handlerName, eventName } = input;
```

Comment:
should this be inputs, plural?
`const { handlerFilename, handlerName, eventName } = inputs;`

like it is in the file `packages/amplify-category-function/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.js` line 25?

Reply:
Yes, indeed. Thank you for your detailed review.